Aspire Budgeting/Views/ErrorBannerView.swift

```diff
@@ -2,17 +2,14 @@
 //  ErrorBannerView.swift
 //  Aspire Budgeting
 //
-//  Created by TeraMo Labs on 11/2/19.
-//  Copyright © 2019 TeraMo Labs. All rights reserved.
-//
 
 import SwiftUI
 
 struct ErrorBannerView: View {
-  var error: GoogleDriveManagerError
+  var error: Error
 
   var body: some View {
-    Text(error.rawValue).font(.custom("Rubik-Light", size: 18)).foregroundColor(.white).opacity(0.6)
+    Text(error.localizedDescription).font(.custom("Rubik-Light", size: 18)).foregroundColor(.white).opacity(0.6)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/d5353e560168880237460fc75a2a6ee44e50449b/Aspire Budgeting/Views/ErrorBannerView.swift#L12"></span>Line should be 100 characters or less: currently 112 characters<br />
<code>line_length</code> <code>ErrorBannerView.swift:12</code></td>
    </tr>
  </tbody>
</table>